classes/Language.php

```diff
@@ -1100,8 +1100,13 @@ public static function downloadXLFLanguagePack($locale, &$errors = array(), $typ
                 curl_setopt($ch, CURLOPT_FOLLOWLOCATION, 0);
                 $raw_file_data = curl_exec($ch);
            
-                if(curl_errno($ch)){
-                   echo 'error:' . curl_error($ch);
+                if (curl_errno($ch)){
+                    throw new \Exception(
```

Comment:
Can you use namespace please :)

Reply:
Hi @PierreRambaud, how would that look like? I copied the Exception throwing from somewhere else.

Reply:
`use Exception;` as the top of the file and remove the `\` =)